src/bit-systems/media-loading.ts

```diff
@@ -176,8 +210,11 @@ function* loadAndAnimateMedia(world: HubsWorld, eid: EntityID, clearRollbacks: C
   removeComponent(world, MediaLoader, eid);
 
   if (media) {
+    const box = getBox(world, eid, media);
+    MediaContentBounds.set(media, box.getSize(new Vector3()));
```

Comment:
Similar to `getBox`, ideally creating Vector3 for each call should be avoided?

Reply:
Updated